src/CppParser/Bootstrap/Bootstrap.cs

```diff
@@ -31,27 +31,30 @@ static string GetSourceDirectory(string dir)
         public void Setup(Driver driver)
         {
             var options = driver.Options;
-            options.LibraryName = "CppSharp";
+            options.GeneratorKind = GeneratorKind.Csharp;
             options.DryRun = true;
-            options.Headers.AddRange(new string[]
-            {
-                "clang/AST/Expr.h",
-            });
-
             options.SetupXcode();
             options.MicrosoftMode = false;
             options.TargetTriple = "i686-apple-darwin12.4.0";
 
-            options.addDefines ("__STDC_LIMIT_MACROS");
-            options.addDefines ("__STDC_CONSTANT_MACROS");
+            var module = options.AddModule("CppSharp");
+
+            module.Defines.add("__STDC_LIMIT_MACROS");
```

Comment:
How does the lower case work when List<>.Add is upper-cased? Does your code compile at all?